### ISSUE TITLE
Poznań GEOPOZ license 

### ISSUE THREAD

**Issue by Asteliks** [osmlab/editor-layer-index#1010]
Well it took a while but now I have a full solution to our license problem mentioned in https://github.com/osmlab/editor-layer-index/pull/957#issuecomment-713275368

Basically after a ton of mails back and fourth I got this replay:
![image](https://user-images.githubusercontent.com/62251475/102101877-79a05a80-3e2b-11eb-9f67-0fab4cb70bb9.png)

> Szanowny Panie,
> 
> w uzgodnieniu z moim Szefem, Zastępcą Dyrektora ZGiKM GEOPOZ ds. SIP, przesyłam Panu nasze stanowisko dotyczące > wykorzystywania ortofotomapy udostępnianej na naszej stronie:
> 
> Uprzejmie informujemy, że do ortofotomapy, udostępnianej na stronie ZGiKM GEOPOZ, nie mają zastosowania przepisy ustawy o prawie autorskim i prawach pokrewnych z dnia 4 lutego 1994 r. tj. z dnia 6 czerwca 2019 r. (Dz.U. z 2019 r. poz. 1231).
Ortofotomapa nie ma charakteru utworu, będącego, zgodnie z ustawą „przejawem działalności twórczej o indywidualnym charakterze”. Jest to ogólnie znany produkt procesu technologicznego, polegającego na utworzeniu kartometrycznego obrazu terenu w formie rastrowej, powstający w wyniku przetworzenia obrazów cyfrowych, odpowiadający rzutowi ortogonalnemu na powierzchnię odniesienia.
Zasady wykorzystania ortofotomapy udostępnionej przez ZGiKM GEOPOZ, zostały opisane w Biuletynie Informacji Publicznej w zakładce „Ponowne wykorzystanie informacji sektora publicznego”. Za spełnienie obowiązku poinformowania o przetworzeniu informacji ponownie wykorzystywanej uznamy jednorazowe powiadomienie ZGiKM GEOPOZ o wykorzystywaniu w sposób ciągły > udostępnionej ortofotomapy do aktualizacji zasobów OpenStreetMap.
> 
> Z poważaniem 
> Roman Dudziak
> 
> Stanowisko ds. Wsparcia Merytoryczno-Organizacyjnego SIP
> 
> Zarząd Geodezji i Katastru Miejskiego GEOPOZ  
ul. Gronowa 20,  
61-655 Poznań  
tel. 61 8271 568 
e-mail: roman.dudziak@geopoz.poznan.pl 
> www.geopoz.poznan.pl 

Basically according to this we need to notify them that we are using their service but have to do this only ones, not every time a user visits our site. 

Now the question is who will notify them? Imho we have two options:
- someone from this repository, preferably a collaborator
- we ask the Polish OSM community to notify them http://openstreetmap.org.pl/kontakt/

This issue solves every Poznań GEOPOZ license problem

**Comment by matkoniecz** [comment]
I see no special requirements, I think that any OSM mapper may notify on behalf of entire OSM Community.

People operating this repo have no special standing if above would be insufficient.

Disclaimer: not a lawyer

**Comment by Asteliks** [comment]
yes and no. Technically any mapper can do that but in case they'll want to contact OSM they will through that mapper, in general that solution is not futureproof in case that mapper no longer has any contact with OSM. That is why I want a person / unit better connected with OSM to be the contact link
